pkg/kubelet/cm/memorymanager/memory_manager_test.go

```diff
@@ -2126,10 +2122,6 @@ func TestGetTopologyHints(t *testing.T) {
 						NUMANodeAffinity: newNUMAAffinity(1),
 						Preferred:        true,
 					},
-					{
-						NUMANodeAffinity: newNUMAAffinity(0, 1),
-						Preferred:        false,
-					},
```

Comment:
Why does this need to be removed? Is there an alternative test to put in its place?